github_app_user_auth/jupyter.py

```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':    
+    from .auth import main
+    
+    main(in_jupyter=True)
```

Comment:
Would it be helpful to have this as a registered console_script entry_point in setup.py?

https://github.com/yuvipanda/github-app-user-auth/blob/f480c0fd708e5b6be7f8d456ad660c8b32e3a33e/setup.py#L16-L20